includes/class-wc-admin-note.php

```diff
@@ -0,0 +1,463 @@
+<?php
+/**
+ * WooCommerce Admin (Dashboard) Notes.
+ *
+ * The WooCommerce admin notes class gets admin notes data from storage and checks validity.
+ *
+ */
+defined( 'ABSPATH' ) || exit;
+
+class WC_Admin_Note extends WC_Data {
+
+	// Note types
+	const E_WC_ADMIN_NOTE_ERROR = 'error';
```

Comment:
Missing comments for all conts here, also all inline comments need to stop with `.`.

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/464/commits/0aaca72b0d2d4adcf7955a8264337ade36232de0